recipes/gsl-lite/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "gsl-lite" %}
+{% set version = "0.32.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/martinmoene/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 134c891b0b0f038d622554faa4040f6d419c534ed18c1b893f4f3ff788515d10
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - make  # [unix]
+
+test:
+  requires:
+    - cmake
+    - {{ compiler('cxx') }}
+    - make  # [unix]
+  files:
+    - test
+
+  commands:
+    - cd test
+    - cmake . -DCMAKE_BUILD_TYPE=Release
+    - cmake --build . --config Release
+    - ./program  # [unix]
```

Comment:
Please, try to add permission to execute this, because it may be the problem in azure
```
chmod +x program  
```

Reply:
This wasn't needed in Linux (CMake+make+gcc already generates an executable with executable permission)... is it really needed for macOS?

Reply:
It is not necessary I was just doing some tests